README.md

```diff
@@ -54,9 +54,11 @@ var myOverlay = new Overlay('myOverlay', {
 	* `.visuallyHideTitle`: Boolean. If you want to provide a different title style, this option will prevent the title span from being added to the overlay. (In this case the title is only used for `aria` labelling) _Default_: false.
 	* `.shaded`: Boolean. Whether to shade the background of the header
 * `modal`: Boolean. Whether the overlay should have modal behaviour or not. Setting this as true will add a translucent shadow between the page and the overlay
+* `fullscreen`: Boolean. If set to true, the overlay will display full screen. This overlay disables scroll on the underlying document and is dismissible with the back button.
 * `compact`: Boolean. If true, the `.o-overlay--compact` class will be added to the overlay that reduces heading font-size and paddings in the content.
 * `src`: String. Either a _url_ from which HTML to populate the overlay can be loaded, or a querySelector string identifying an element from which the textContent should be extracted.
 * `html`: String or HTMLElement.  Raw HTML (cannot be set declaratively)
+* `class`: String. The custom classes to apply to to the overlay e.g. `o-overlay--my-modifier`.
```

Comment:
Do we do this with other components? I'm under the impression that we provide basic styling for most things if the product using the component includes the full range of origami classes, and then it applies classes of its own?

Reply:
Yeah usually, but you can also add custom classes to the html so you can apply your own selectors against specific instances if you need to. Here the outer `o-overlay` dom node isn't accessible. I'm not sure how popular that is and wouldn't recommend if there was another approach, but don't see any reason to discourage it.